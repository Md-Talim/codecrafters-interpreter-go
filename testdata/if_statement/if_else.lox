// Test case 1
if (true) print "if branch"; else print "else branch";

// Test case 2
var age = 21;
if (age > 18) print "adult"; else print "child";

// Test case 3
if (false) {
  print "if block";
} else print "else statement";

if (false) print "if statement"; else {
  print "else block";
}