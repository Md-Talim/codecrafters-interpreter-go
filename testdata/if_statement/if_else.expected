if branch
adult
else statement
else block